src/code.c

```diff
@@ -212,7 +212,7 @@ void SET_ENDLINE_BODY(Obj body, Obj val)
 ** Fill in filename and line of a statement, checking we do not overflow
 ** the space we have for storing information
 */
-Stat fillFilenameLine(Int fileid, Int line, Int size, Int type)
+static Stat fillFilenameLine(Int fileid, Int line, Int size, Int type)
 {
```

Comment:
I think @ChrisJefferson added this. Of course I can change it, but I actually would prefer to just refact that code to use a bitfield for the header of a stat, instead of this helper function; and then also fix the alignment of stat's on 32bit platforms.